onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -203,6 +203,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "The value to be filled in the attention mask. Default value is -10000.0f",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("norm_factor",
```

Comment:
why not define scale as parameter?
If not specified, we will use 1/sqrt(head_size) as scale.

Reply:
I'm good with this definition, but I think it would be a bit more intuitive to directly get the norm_factor from Div node in onnx graph.

Reply:
I prefer "scale" since it is a term used in original paper (https://arxiv.org/pdf/1706.03762.pdf). In section of `Scaled Dot Production Attention`, `To counteract this effect, we scale the dot products by 1/sqrt(Dk).`.  So it is used to scale but not for normalization.

Reply:
makes sense. will change to scale